src/server/scripts/Northrend/Ulduar/HallsOfLightning/boss_ionar.cpp

```diff
@@ -226,9 +226,9 @@ class boss_ionar : public CreatureScript
             // Splitted
             if (!me->IsVisible())
             {
-                if (uiSplitTimer <= uiDiff)
+                if (uiSplitTimer <= Seconds(uiDiff))
```

Comment:
And this type of checking should also be replaced with task/event scheduling.

Reply:
Update diff is in milliseconds, not seconds